tests/legacy-cli/e2e/tests/i18n/extract-ivy.ts

```diff
@@ -38,13 +38,12 @@ export default async function () {
   if (getGlobalVariable('argv')['ng-snapshots']) {
     localizeVersion = require('../../ng-snapshot/package.json').dependencies['@angular/localize'];
   }
-  await installPackage(localizeVersion);
+  await ng('add', localizeVersion, '--skip-confirmation');
```

Comment:
I doubt that this will work for snapshots.

Reply:
Is there a way to check that?

Reply:
`ng add` should work with github URLs.  You can verify by getting the value from the snapshot file in the repo and running `ng add` locally with that value.

Reply:
The snapshot tests will fail on main if left in place. You can disable the snapshot tests here for now until `ng add` is updated in the future to use the package managers directly instead of the `pacote` package.

Reply:
Ok I skipped the test as done in other places. Let me know if that's OK